kymatio/scattering1d/scattering1d.py

```diff
@@ -16,7 +16,7 @@
 from .filter_bank import calibrate_scattering_filters
 
 
-def compute_minimum_support_to_pad(T, J, Q, criterion_amplitude=1e-3,
+def compute_minimum_support_to_pad(J, T, Q, criterion_amplitude=1e-3,
```

Comment:
not sure if this makes sense semantically. padding is about `T` first and foremost

Reply:
you're right. I was on the verge about this one. will fix

Reply:
fixed in 805a9f1